s/stack/stack_v1.8.0_UBI_8.5.sh

```diff
@@ -0,0 +1,62 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : github.com/go-stack/stack
+# Version       : v1.8.0
+# Source repo   : https://github.com/go-stack/stack.git
+# Tested on     : RHEL 8.4
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Amit Baheti (aramswar@in.ibm.com)
+#
+# Disclaimer    : This script has been tested in root mode on given
+# ==========    platform using the mentioned version of the package.
+#               It may not work as expected with newer versions of the
+#               package and/or distribution. In such case, please
+#               contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+PACKAGE_NAME=github.com/go-stack/stack
+#Setting the default version v1.8.0
+PACKAGE_VERSION=${1:-v1.8.0}
+PACKAGE_PATH=https://github.com/go-stack/stack.git
+
+if ! command -v git &> /dev/null
+then
+    yum install -y git
+fi
+# Install Go and setup working directory
+if ! command -v go &> /dev/null
+then
+    yum install -y golang
+fi
+
+mkdir -p /root/output
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+export GOPATH="$(go env GOPATH)"
+export PATH=$GOPATH/bin:$PATH
+export GO111MODULE=on
+
+echo "Building $PACKAGE_PATH with $PACKAGE_VERSION"
+rm -rf $GOPATH/src/github.com/stack
+mkdir -p $GOPATH/src/github.com/ && cd $GOPATH/src/github.com/
```

Comment:
Just a suggestion,
Anyway you are using `$GOPATH/src/github.com`.
So can you clone the package into `$GOPATH/src/github.com/go-stack` ?

Rest looks good.

Reply:
@kandarpamalipeddi The scripts changes the directory to github.com & then it git clones the repo which creates "stack" folder inside.

Reply:
My point is when we are cloning the repo to the GOPATH, 
Can't we replace below lines :

```
mkdir -p $GOPATH/src/github.com/ && cd $GOPATH/src/github.com/
```
with
```
mkdir -p $GOPATH/src/github.com/go-stack && cd $GOPATH/src/github.com/go-stack
```

This is not a rule, its just a best practice.

Reply:
Added the folder